internal/provider/configuration/provider.go

```diff
@@ -0,0 +1,88 @@
+package configuration
+
+import (
+	"fmt"
+	"io"
+	"strings"
+
+	"github.com/kyma-project/kyma-environment-broker/common/runtime"
+	"gopkg.in/yaml.v2"
+)
+
+type ProviderSpec struct {
+	data dto
+}
+
+type regionDTO struct {
+	DisplayName string   `yaml:"displayName"`
+	Zones       []string `yaml:"zones"`
+}
+
+type providerDTO struct {
+	Regions map[string]regionDTO `yaml:"regions"`
+}
+
+type dto map[runtime.CloudProvider]providerDTO
+
+func NewProviderSpec(r io.Reader) (*ProviderSpec, error) {
+	data := &dto{}
+	d := yaml.NewDecoder(r)
+	err := d.Decode(data)
+	return &ProviderSpec{
+		data: *data,
+	}, err
+}
+
+func (p *ProviderSpec) RegionDisplayName(cp runtime.CloudProvider, region string) string {
```

Comment:
Following our direct discussion about missing display names - we could handle it here (returning region as "default").